cluster.go

```diff
@@ -189,12 +189,12 @@ type ClusterConfig struct {
 	// set to 10.0.0.1 which is what will be used to connect to.
 	IgnorePeerAddr bool
 
-	// If DisableInitialHostLookup then the driver will not attempt to get host info
+	// If DisableHostLookup then the driver will not attempt to get host info
 	// from the system.peers table, this will mean that the driver will connect to
 	// hosts supplied and will not attempt to lookup the hosts information, this will
 	// mean that data_centre, rack and token information will not be available and as
 	// such host filtering and token aware query routing will not be available.
-	DisableInitialHostLookup bool
+	DisableHostLookup bool
```

Comment:
I don't think this is worth breaking applications. I agree that removing the `Initial` word from the name does make sense but it's really not that big of a deal.

Reply:
At the very least we would have to keep both flags and map it to a single internal flag so that we can do a "safe" rename of the flag without breaking user applications (and deprecate the old one).